docs/source/_notebooks/keras-text-classifiers.rst

```diff
@@ -0,0 +1,4124 @@
<span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 87.83%); opacity: 0.84" title="0.002">to</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 88.13%); opacity: 0.84" title="0.002">know</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 92.41%); opacity: 0.82" title="0.001">this</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 79.72%); opacity: 0.88" title="0.004">wasn&#x27;t</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 80.55%); opacity: 0.87" title="0.003">going</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 88.32%); opacity: 0.83" title="0.002">to</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 81.43%); opacity: 0.87" title="0.003">work</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 80.89%); opacity: 0.87" title="0.003">out</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 87.99%); opacity: 0.84" title="0.002">and</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 87.03%); opacity: 0.84" title="0.002">his</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 78.29%); opacity: 0.88" title="0.004">performance</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 75.41%); opacity: 0.90" title="0.005">was</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 73.69%); opacity: 0.91" title="0.005">quite</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 86.69%); opacity: 0.84" title="0.002">&lt;OOV&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 69.50%); opacity: 0.94" title="0.006">so</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 67.84%); opacity: 0.95" title="0.007">all</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 65.70%); opacity: 0.96" title="0.008">you</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 77.26%); opacity: 0.89" title="0.004">madison</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 60.00%); opacity: 1.00" title="0.009">fans</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 73.31%); opacity: 0.91" title="0.005">give</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 82.62%); opacity: 0.86" title="0.003">this</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 70.33%); opacity: 0.93" title="0.006">a</span><span style="opacity: 0.80"> </span><span
```

Comment:
Maybe it would be more clear if we printed only the layers which we use for explanation.